puppet-syntax.gemspec

```diff
@@ -21,5 +21,7 @@ Gem::Specification.new do |spec|
   spec.add_dependency "rake"
 
   spec.add_development_dependency "rspec"
+  spec.add_development_dependency "pry"
```

Comment:
I dont think they need to be a separate PR but they should probably be separate commits within this PR.